### ISSUE TITLE
Highlighting of TODO is incorrect when preceded by whitespace

### ISSUE THREAD

**Issue by TMVector** [omsharp/BetterComments#12]
### Installed product versions
- Visual Studio: 2015 Community
- This extension: 1.6.24

### Description
When the `Color only the 'Todo' keyword (Task Comments)` option is selected, only the first four characters of a TODO comment are highlighted correctly. For example (^ indicates the task highlight):

```
//TODO create a banana smoothie
  ^^^^
// TODO drink that delicious smoothie
  ^^^^
//  TODO wipe off your milk moustache
  ^^^^
```

### Steps to recreate
1. Enable the  `Color only the 'Todo' keyword (Task Comments)` option in Tools -> Options -> Better Comments -> Font Options
2. Create a TODO comment with a space between the double slash and the TODO word (I've only tested this in a C# file)

### Current behavior
Only the first four characters are highlighted, including whitespace, and so the end of the task word is not highlighted.

### Expected behavior
The task word (Todo), the whole task word, and nothing but the task word should be highlighted.

*I would be happy to create a fix/PR, but will probably be too busy until June because of finals.*

**Comment by omsharp** [comment]
I'll address this issue as soon as I can. But if you got to it before I do, I'll be glad to accept your Pull Request.

**Comment by omsharp** [comment]
Fixed in v1.6.26

**Comment by omsharp** [comment]
Fixed in v1.6.26
